### PR TITLE
Ladder peeking and grenade throwing

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request

This PR adds the ability to see down and up ladders and also see down and up stairs by Alt Clicking the multiz structure 
This PR also adds the ability to thrown items up and down ladders by clicking on the ladder with the item
This PR has been heavily tested for ways to break the code and mess with planes
## Why It's Good For The Game

This completely destroys the z-level transverse camping that was possible before, in which you could wait for someone, and instantly open fire and kill them before they could know what was going on. Also the new functionality for  items

## Changelog
:cl:
add: You can now see down and up ladders, and up and down stairs by alt clicking the zlevel change structure, whether it be a ladder or a stair
add: You can now throw items up and down ladders by clicking on the ladder with the item inhand (grenades blow up)
/:cl:

### CODE REVIEW

**Review by Hopekz — APPROVED** [review]
Approving the idea of this PR fitting into the codebase concept-wise.
Still needs code review.

**Comment by MacBlaze1** [comment]
dont merge yet I still need to add throwing nades if I can

**Review by genessee596 — APPROVED** [review]
This PR gets a second Delegate stamp of approval from yours truly.

Concept is good, makes ladders seem more like ladders and not portals that players alone (and not items) can traverse.

Still needs code review.

File: code/modules/multiz/structures.dm
Comment (TheShown911) [inline code/modules/multiz/structures.dm:113]: Don't see why this should only work with nades.

Comment (TorinoFermic) [reply code/modules/multiz/structures.dm:113]: It would be nice to let players throw down other stuff like guns or money, etc.

**Review by TorinoFermic — APPROVED** [review]

File: code/modules/mob/living/carbon/carbon_defines.dm
Comment (TheShown911) [inline code/modules/mob/living/carbon/carbon_defines.dm:29]: TRUE and FALSE would work nicer for this, but eh who cares

File: code/modules/multiz/structures.dm
Comment (TheShown911) [inline code/modules/multiz/structures.dm:118]: Span macros would be better for this

File: code/modules/mob/living/carbon/carbon_defines.dm
Comment (mattroks101) [reply code/modules/mob/living/carbon/carbon_defines.dm:29]: I care. Please use TRUE and FALSE.

File: code/modules/multiz/structures.dm
Comment (mattroks101) [reply code/modules/multiz/structures.dm:118]: I also care about this, please use span macros.

Comment (MacBlaze1) [reply code/modules/multiz/structures.dm:118]: what are spam macros

Comment (MacBlaze1) [reply code/modules/multiz/structures.dm:118]: I understand they condense, but uh

Comment (TorinoFermic) [reply code/modules/multiz/structures.dm:118]: Sort of SPAN_WARNING("text here")

File: code/modules/mob/living/carbon/carbon_defines.dm
Comment (MacBlaze1) [reply code/modules/mob/living/carbon/carbon_defines.dm:29]: done

File: code/modules/multiz/structures.dm
Comment (MacBlaze1) [reply code/modules/multiz/structures.dm:118]: done

File: code/modules/multiz/structures.dm
Comment (MacBlaze1) [reply code/modules/multiz/structures.dm]: done

**Review by TheShown911 — APPROVED** [review]

**Comment by genessee596** [comment]
Got approved by 2 maintainers and 2 delegates.

Gonna go ahead and merge this now.

**Pull request merged**
